cmd/goProbe/goProbe.go

```diff
@@ -59,8 +59,15 @@ func main() {
 	}
 
 	// Initialize logger
-	err = logging.Init(logging.LevelFromString(config.Logging.Level), logging.Encoding(config.Logging.Encoding),
+	loggerOpts := []logging.Option{
 		logging.WithVersion(appVersion),
+	}
+	if config.Logging.Destination != "" {
+		loggerOpts = append(loggerOpts, logging.WithFileOutput(config.Logging.Destination))
```

Comment:
How about having `logging.WithFileOutput()` treat an empty string as null-op? Then you can save yourself the whole trouble with building up an option array and conditionally calling the option?

Reply:
Alright then, let's keep it as is.